repository/src/main/java/org/alfresco/repo/node/db/DbNodeServiceImpl.java

```diff
@@ -3203,13 +3203,16 @@ else if (!nodeDAO.hasNodeAspect(childNodeId, ContentModel.ASPECT_AUDITABLE))
                     // Invoke policy behaviour
                     invokeBeforeUpdateNode(parentNodeRef);
 
+                    Map<QName, Serializable> propertiesBefore = nodeDAO.getNodeProperties(parentNodeId);
                     // Touch the node; it is cm:auditable
                     boolean changed = nodeDAO.setModifiedProperties(parentNodeId, modifiedDate, modifiedByToPropagate);
+                    Map<QName, Serializable> propertiesAfter = nodeDAO.getNodeProperties(parentNodeId);

```

Comment:
Reminder to move this within the `if (changed) ...` block to avoid unnecessarily querying the database, as mentioned by @jamalkm